### PR TITLE
[kubevirt-presubmits]: update the sig-monitoring job configuration

### PR DESCRIPTION
**Pull request by enp0s3**
remove the optional flag so the lane will always run upon
monitoring code change. Also make the report visible since
not all the contributors are familiar with the Prow dashboard.

Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,9 +17,7 @@ require (
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.13.0
 	github.com/prometheus/client_golang v1.11.0
-	github.com/shurcooL/githubv4 v0.0.0-20210725200734-83ba7b4c9228 // indirect
```
Comment (dhiller) [inline go.mod:20]: This looks like an unrelated change?

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml [hunk 2]
```diff
@@ -1867,7 +1867,6 @@ presubmits:
       preset-dind-enabled: "true"
       preset-docker-mirror-proxy: "true"
     name: pull-kubevirt-e2e-k8s-1.22-sig-monitoring-0.49
-    optional: true
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml:1870]: Do we really want to do this for the 0.49 lane, not sure we have enough data for this? Or is this required as we are going to backport some stuff here?

File: go.mod (hunk 1 above)
Comment (enp0s3) [reply go.mod]: @dhiller Correct. Fixed

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml (hunk 2 above)
Comment (enp0s3) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml:1870]: @dhiller Currently we don't have any PR related to monitoring that has to be backported. I was only thinking about a possibility for that to happen. I think that in general if that will happen and we will keep it optional in 0.49, it will be tested on main but not tested on the 0.49 which is inconsistent, WDYT?

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml:1870]: If you remove the `optional: true` it will still be tested but the merge will not be gated by tide should the lane fail.

Comment (enp0s3) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.49.yaml:1870]: @dhiller I see, and since the report will appear, the reviewer will notice the failure of the lane. So lets keep optional then. Thank you for the brainstorm!

**Review by dhiller — APPROVED** [review]
/approve
